src/app/modules/ix-forms/components/ix-select/ix-select.component.ts

```diff
@@ -37,21 +42,18 @@ export class IxSelectComponent implements ControlValueAccessor, OnChanges {
   }
 
   ngOnChanges(): void {
-    if (this.options) {
+    if (!this.options) {
+      this.hasErrorInOptions = true;
+    } else {
       this.opts$ = this.options.pipe(
```

Comment:
Set `hasErrorInOptions = false;` here so the options load when correct value is assigned to `this.options`.

Reply:
@Raz-Dva Please fix the requested issue

Reply:
@[RehanY147](https://github.com/RehanY147) Ok